plugins/source/gitlab/codegen/recipes/users.go

```diff
@@ -0,0 +1,29 @@
+package recipes
+
+import (
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func Users() []*Resource {
+	resources := []*Resource{
+		{
+			Service:    "users",
+			SubService: "users",
+			PKColumns:  []string{"base_url", "id"},
+			Struct:     &gitlab.User{},
+			SkipFields: []string{"LastActivityOn"},
+			ExtraColumns: []codegen.ColumnDefinition{
+				{
+					Name:          "last_activity_on",
+					Type:          schema.TypeJSON,
+					Resolver:      `schema.PathResolver("LastActivityOn")`,
+					IgnoreInTests: true,
```

Comment:
1. `LastActivityTime` should be. `schema.TypeTimestamp` 9and can be achieved via type transformer)
2. Maybe there's no need to ignore it, just set it manually in test after faker?

Reply:
fixed